java/semantickernel-api/src/main/java/com/microsoft/semantickernel/KernelConfig.java

```diff
@@ -1,186 +1,11 @@
 // Copyright (c) Microsoft. All rights reserved.
 package com.microsoft.semantickernel;
 
-import com.microsoft.semantickernel.ai.embeddings.EmbeddingGeneration;
-import com.microsoft.semantickernel.chatcompletion.ChatCompletion;
-import com.microsoft.semantickernel.textcompletion.TextCompletion;
-
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Locale;
-import java.util.Map;
-import java.util.function.Function;
-
-import javax.annotation.Nullable;
-
+// Currently the kernel config does not have and function within Java
```

Comment:
Should it be marked as deprecated?

Reply:
I would not describe it as deprecated, stuff will go in here in the future, just for now there is nothing